packages/cf-builder-card/src/cardReducer.js

```diff
@@ -1,10 +1,12 @@
-const ActionTypes = require('./CardActionTypes');
+// @flow
 
-const initialState = {};
+import type {CardAction, CardState} from './CardTypes';
```

Comment:
You know it's weird to be concerned about class properties while being okay with imports right? Imports are less standardized than class properties.